mergify_engine/tests/functional/test_github_async_client.py

```diff
@@ -68,3 +68,25 @@ async def test_github_async_client(self):
             await client.item(f"{url}/10000000000")
 
         self.assertEqual(404, ctxt.exception.response.status_code)
+
+    async def test_github_async_client_with_owner_id(self):
+
+        rules = {
+            "pull_request_rules": [
+                {
+                    "name": "fake PR",
+                    "conditions": ["base=master"],
+                    "actions": {"merge": {}},
+                }
+            ]
+        }
+
+        await self.setup_repo(yaml.dump(rules))
+        p, _ = await self.create_pr()
+
+        client = github.aget_client(self.o_integration.login, self.o_integration.id)
```

Comment:
since both login and ids are passed, I guess only one is used?
do we want to test only `id` being passed there?